shapeai/.env.example

```
@@ -0,0 +1,15 @@
+# Mobile (apps/mobile/.env)
+EXPO_PUBLIC_API_URL=http://localhost:3000
+EXPO_PUBLIC_SUPABASE_URL=https://your-project.supabase.co
+EXPO_PUBLIC_SUPABASE_ANON_KEY=your-anon-key
+EXPO_PUBLIC_REVENUECAT_KEY=appl_xxx
+
+# API Gateway (services/api-gateway/.env)
+PORT=3000
+DATABASE_URL=postgresql://user:password@localhost:5432/shapeai
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the PORT key

(UnorderedKey)